src/Workspaces/Core/Portable/Workspaces.csproj

```diff
@@ -257,6 +257,7 @@
     <InternalsVisibleToMonodevelop Include="MonoDevelop.Refactoring.Tests" />
     <InternalsVisibleToMonodevelop Include="MonoDevelop.CSharpBinding.Tests" />
     <InternalsVisibleToMonodevelop Include="MonoDevelop.VBNetBinding.Tests" />
+    <InternalsVisibleToProjectSystem Include="Microsoft.VisualStudio.ProjectSystem.FSharp" />
```

Comment:
@pilchie We have to live with public APIs for many years.  The design work to do that is going to take longer than just making the API we have currently implemented public.

At some point we need to sit down and look at the internal APIs that F# and TypeScript deened on and design what we want the public APIs to look like.

At least this way the exposure is controlled.